### ISSUE TITLE
discourge and add warnning for vbox driver if there are other healthier options available

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#11836]
some users might have set config to use vbox driver in the past and minikube respects it and wont change to better drivers
in these cases we should show a Box UI warnining that there are better optionns avilable and here are the commandns to use it

**Comment by afbjorklund** [comment]
I don't think we should complain to users using the VirtualBox driver (or the Docker runtime), even if there are newer alternatives.
The "native virtualization" and the containerd/buildkitd runtime has yet much to prove, before they replace the traditional options.
